src/egl/drivers/dri2/egl_dri2.h

```diff
@@ -59,6 +59,7 @@
 #include <system/window.h>
 #include "platform_android_gralloc_drm.h"
 #include <hardware/gralloc1.h>
```

Comment:
ok now I got it, this is to be merged together with changes in gralloc1.0 branch